recipes/otrobopt/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "0.1" %}
+
+package:
+  name: otrobopt
+  version: {{ version }}
+
+source:
+  fn: otrobopt-{{ version }}.tar.gz
+  url: https://github.com/openturns/otrobopt/archive/v{{ version }}.tar.gz
+  sha256: 7ea2d3e321a2aae80d7361074627029490202ad33a03b557014b38393bb13c9a
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - python
+    - cmake
+    - swig
+    - openturns
+
+  run:
+    - python
+    - openturns
+
+test:
+  imports:
+    - otrobopt
```

Comment:
@jschueller I suggest you add the following command "test" to that help debug issues in the future:

```yaml
    commands:
        - conda inspect linkages -n _test otrobopt  # [not win]
        - conda inspect objects -n _test otrobopt  # [osx]
```

They are not really package tests but you can inspect how the package linking to other libs and the objects in them.